plugins/plugin-run-script/plugin.js

```diff
@@ -1,6 +1,11 @@
 const execa = require('execa');
 const npmRunPath = require('npm-run-path');
 
+const CLEAR_SEQUENCES = [
+  '\x1Bc',
```

Comment:
love the idea behind this PR, but why remove `\u001bc`?

Reply:
`"\u001bc"` is the same as `"\x1Bc"`

See: https://runkit.com/nonzero/u001bc-x1bc

Reply:
TIL! Thanks for the link